project-base/composer.json

```diff
@@ -139,9 +139,9 @@
         "auto-scripts": {
             "php phing clean": "script",
             "shopsys:domains-urls:configure": "symfony-cmd",
-            "ckeditor:install --clear=skip --release=full --tag=4.5.11": "symfony-cmd",
-            "security-checker security:check": "script"
-        }
+            "ckeditor:install --clear=skip --release=full --tag=4.5.11": "symfony-cmd"
+        },
+        "security-check": "security-checker security:check"
```

Comment:
Should not it be part of `auto-scripts` too?

```
        "auto-scripts": {
            "php phing clean": "script",
            "shopsys:domains-urls:configure": "symfony-cmd",
            "ckeditor:install --clear=skip --release=full --tag=4.5.11": "symfony-cmd"
            "composer security-check": "script"
        },
        "security-check": "security-checker security:check"
```

Reply:
in project-base it's separate intentionally as new projects very often remove the security check from auto-scripts and add it as a check on CI performed every midnight.
Because it's better to run a security check on a time basis instead of push (projects can be without active development for a while, but check for known security issues should be performed anyway) and to prevent blocking of CI server for sending many requests.

Reply:
Well, what about projects which do not have the cron set? Now their build will be passing even on security issue.

I do mind we are not only changing the scipt which is checking it, but also change the bahavior of checking - which is really hidden in it.

Reply:
changed to run only after an update (require), as we discussed in ed8fbe8